test/writeIndex.js

```
@@ -8,29 +8,59 @@ import {
 import writeIndex from '../src/utilities/writeIndex';
 import codeExample from './codeExample';
 
+const readFileByPath = (filePath) => {
+  return fs.readFileSync(filePath, 'utf8');
+};
+
+const removeFileByPath = (filePath) => {
+  // eslint-disable-next-line no-empty
+  try {
```

Whats the reason for swallowing an error?